dashboard/src/modules/components/SidebarComponent/index.jsx

```diff
@@ -1,12 +1,59 @@
-import React from "react";
-import { PageSidebar } from "@patternfly/react-core";
-import NavItems from "../NavbarComponent";
-import { useSelector } from "react-redux";
+import { Nav, NavGroup, NavItem, PageSidebar } from "@patternfly/react-core";
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useLocation, useNavigate } from "react-router-dom";
 
+import { menuOptions } from "./sideMenuOptions";
+import { setActiveItem } from "actions/sideBarActions";
+
+const Menu = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { pathname } = useLocation();
+
+  const activeItem = useSelector((state) => state.sidebar.activeMenuItem);
+  const onSelect = (result) => {
+    dispatch(setActiveItem(result.itemId));
+  };
+
+  useEffect(() => {
+    if (pathname) {
+      const currPath = pathname.split("/").at(-1);
+      dispatch(setActiveItem(currPath));
+    }
+  }, [dispatch, pathname]);
+  return (
+    <Nav onSelect={onSelect}>
+      {menuOptions.map((item, index) => {
+        return (
```

Comment:
If your arrow-function body doesn't contain any statements other than `return`, consider using an expression instead of a block -- i.e., instead of lines 27-28 here, I think you could just use
```
      {menuOptions.map((item, index) => (
```
Given that you have three, nested arrow-functions with that form, here, I think you'll find that using expressions like this makes the code simpler (i.e., a few lines shorter with a little less indentation).